composer.json

```diff
@@ -13,13 +13,14 @@
         "ext-json": "*",
         "ext-pdo": "*",
         "ext-zip": "*",
-        "beberlei/assert": "2.x",
+        "beberlei/assert": "^2.9",
```

Comment:
The requirements list should not change. The only new thing is league/csv and this is a result of the new module so not really required here.

Reply:
Ok, but when i run `composer install` thoes libraries was in the higher version. I only change it in `composer.json` to current version.